<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestao_estoque</class>
 <widget class="QDialog" name="fm_gestao_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_produtos">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>741</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novo_produto">
    <attribute name="title">
     <string>Novo produto</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>134</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>R$ compra:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_quantidade_produto"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Quantidade:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_valor_venda_produto"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>R$ venda:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_valor_compra_produto"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>721</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Código de barras:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_id_produto">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Descrição:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_descricao_produto"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>220</y>
       <width>311</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_salvar_produto">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancelar_produto">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>134</y>
       <width>361</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>id_fornecedor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_fornecedor"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestao_produtos">
    <attribute name="title">
     <string>Gestão de produtos</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Perquisar por:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>23</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_id_produto">
         <property name="text">
          <string>Código</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_descricao_produto">
         <property name="text">
          <string>Descrição</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>201</x>
       <y>23</y>
       <width>531</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Pesquisar:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pesquisar_produto"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pesquisar_produto">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>344</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>R$ compra:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_valor_compra_produto_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Quantidade:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>R$ venda:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_quantidade_produto_edit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_valor_venda_produto_edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>721</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_salvar_produto_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Salvar alteração</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_excluir_produto">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancelar_produto_Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Decrição:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_descricao_produto_Edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_id_produto_edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget_produtos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>731</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>343</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Fornecedor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_fornecedor_edit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_fechar">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>482</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget_produtos</tabstop>
  <tabstop>pushButton_fechar</tabstop>
  <tabstop>lineEdit_id_produto</tabstop>
  <tabstop>lineEdit_descricao_produto</tabstop>
  <tabstop>lineEdit_quantidade_produto</tabstop>
  <tabstop>lineEdit_valor_compra_produto</tabstop>
  <tabstop>lineEdit_valor_venda_produto</tabstop>
  <tabstop>pushButton_salvar_produto</tabstop>
  <tabstop>pushButton_cancelar_produto</tabstop>
  <tabstop>radioButton_id_produto</tabstop>
  <tabstop>radioButton_descricao_produto</tabstop>
  <tabstop>lineEdit_pesquisar_produto</tabstop>
  <tabstop>pushButton_pesquisar_produto</tabstop>
  <tabstop>tableWidget_produtos</tabstop>
  <tabstop>lineEdit_id_produto_edit</tabstop>
  <tabstop>lineEdit_descricao_produto_Edit</tabstop>
  <tabstop>lineEdit_quantidade_produto_edit</tabstop>
  <tabstop>lineEdit_valor_compra_produto_edit</tabstop>
  <tabstop>lineEdit_valor_venda_produto_edit</tabstop>
  <tabstop>pushButton_salvar_produto_edit</tabstop>
  <tabstop>pushButton_excluir_produto</tabstop>
  <tabstop>pushButton_cancelar_produto_Edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
